### ISSUE TITLE
Peer dependencies issue in @walletconnect/heartbeat 1.2.0

### ISSUE THREAD

**Issue by adriencohen** [WalletConnect/walletconnect-utils#50]
Hello,

Installing `@walletconnect/heartbeat: 1.2.0` generates the 2 following peerDependencies error with yarn 3:

```
➤ YN0002: │ @walletconnect/heartbeat@npm:1.2.0 doesn't provide @types/node (p37663), requested by ts-node
➤ YN0002: │ @walletconnect/heartbeat@npm:1.2.0 doesn't provide typescript (pd3167), requested by ts-node
```

It looks like there is a difference between the `package.json` dependencies on this repo  and the one in the published package

[From repo:](https://github.com/WalletConnect/walletconnect-utils/blob/master/misc/heartbeat/package.json)
```
  "dependencies": {
    "@walletconnect/events": "^1.0.1",
    "@walletconnect/time": "^1.0.2",
    "tslib": "1.14.1"
  },
```

[From npm:](https://cdn.jsdelivr.net/npm/@walletconnect/heartbeat@1.2.0/package.json)
```
  "dependencies": {
    "@walletconnect/events": "^1.0.1",
    "@walletconnect/time": "^1.0.2",
    "chai": "^4.3.7",
    "mocha": "^10.2.0",
    "ts-node": "^10.9.1",
    "tslib": "1.14.1"
  }
```

Can you take a look at it please ?

**Comment by beeb** [comment]
This is quite annoying. Why do you have a runtime dependency on tslib in several packages of this repo? Maybe I'm missing something but this doesn't look like something the consumers would require at runtime.

**Comment by mlshv** [comment]
having the same issue with this. this is a major blocker for me since [it bloats vercel serverless function bundle](https://github.com/orgs/vercel/discussions/103#discussioncomment-5377806) which makes it impossible to deploy my app

**Comment by krisgrm** [comment]
having same issues here.. is it possible to override dependency's depdency list to force some package to be dev dependency?

**Comment by bkrem** [comment]
Fixed in `heartbeat@1.2.1` and [2.7.0 for the monorepo pkgs](https://github.com/WalletConnect/walletconnect-monorepo/releases/tag/2.7.0)
